user-guide/Standard_Apps/EPM/EPM_VSAT/VSAT_EPM_Topology.md

```diff
@@ -0,0 +1,68 @@
+---
+uid: VSAT_EPM_Topology
+---
+
+# EPM VSAT topology
+
+The EPM topology navigation menu can be used to access the collector elements' main KPIs.
+
+![Topology](~/user-guide/images/VSAT_EPM_Topology.png)
```

Comment:
Maybe we should add the DataMiner version and solution version for this screenshot, possibly in a figure caption below it, which can be configured like this: 
```suggestion
![Topology](~/user-guide/images/VSAT_EPM_Topology.png)<br>
*figure caption*
```

This way, if the screenshot gets outdated, users will know why it's different from what they see in their more recent system. The same could also apply for other update-sensitive screenshots in this PR.